src/celeritas/field/CylMapField.hh

```diff
@@ -52,9 +52,9 @@ class CylMapField
     // Shared constant field map
     FieldParamsRef const& params_;
 
-    NonuniformGrid<real_type> const grid_r_;
-    NonuniformGrid<real_type> const grid_phi_;
-    NonuniformGrid<real_type> const grid_z_;
+    NonuniformGrid<float> const grid_r_;
+    NonuniformGrid<float> const grid_phi_;
+    NonuniformGrid<float> const grid_z_;
```

Comment:
Instead of hardcoding `float`, add `using cylmap_real_type = float;` to the FieldData header, and locally to each class `using real_type = cylmap_real_type;`